### PR TITLE
Add foreman_azure_rm 2.1.x features documentation

### PR DESCRIPTION
**Pull request by apuntamb**
For foreman-2.1, `foreman_azure_rm` has new features in it's latest version `2.1.x`.

### CODE REVIEW

File: plugins/foreman_azure/2.1.x/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+layout: plugin
+title: Foreman AzureRm 2.1.x Manual
+version: 2.1.x
+# uncomment for older versions than stable or nightly
+#robots: noindex
```
Comment (tbrisker) [inline plugins/foreman_azure/2.1.x/index.md:6]: Should this be uncommented for the older versions?

File: plugins/foreman_azure/2.1.x/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+layout: plugin
+title: Foreman AzureRm 2.1.x Manual
+version: 2.1.x
```
Comment (tbrisker) [inline plugins/foreman_azure/2.1.x/index.md:4]: I think you'll need to also add this version to the plugin index

Comment (apuntamb) [reply plugins/foreman_azure/2.1.x/index.md:4]: done.

File: plugins/foreman_azure/2.1.x/index.md (hunk 1 above)
Comment (apuntamb) [reply plugins/foreman_azure/2.1.x/index.md:6]: done.

File: plugins/foreman_azure/2.1.x/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+* You can now fill in your resource group, size of the VM you want to provision, username and password or sshkey with which the user can login to the provisioned machine.
+
+* Now, you can also select additional disks for the host. Note that the maximum number of these disks depends on the VM Size selected. Please refer to MSFT documentation for more details.
+
+* In addition to that, you can provide any custom script or command to be executed during the VM creation process.
+
+* In the Operating Systems tab, you must specify the Architecture, OS and the relevant Image that you have created under the compute resource. Although, setting up Root Password is not mandatory.
+
+* Last specification is for the Network Interface. You should select a domain defined under Foreman and then select Azure subnet and Public or Private IP format. Now, you can also select multiple interfaces for your host. The maximum number of these interfaces depends on the VM Size selected. Please refer to MSFT documentation for more details.
```
Comment (apuntamb) [inline plugins/foreman_azure/2.1.x/index.md:56]: @ShimShtein could you take a look at the `usage` section of the new version?

File: plugins/foreman_azure/2.1.x/index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    $ gem 'foreman_azure_rm'
+
+Then run `bundle install` from the same directory
+
+# 3. Configuration
+
+Please refer to our documentation [Github](https://github.com/theforeman/foreman_azure_rm#foreman-azurerm-plugin)
+
+# 4. Usage
+* Creating the Compute Resource is created requires using four credentials: Client ID, Client Secret, Subscription ID and Tenant ID.
```
Comment (ShimShtein) [inline plugins/foreman_azure/2.1.x/index.md]:
```suggestion
* Creation of the Compute Resource requires using of four credentials: Client ID, Client Secret, Subscription ID and Tenant ID.
```

**Review by ShimShtein — CHANGES_REQUESTED** [review]

File: plugins/foreman_azure/2.1.x/index.md (hunk 4 above)
Comment (apuntamb) [reply plugins/foreman_azure/2.1.x/index.md]: done

**Comment by ShimShtein** [comment]
This looks better, ACK from me.

**Comment by apuntamb** [comment]
@tbrisker Can we get this in? :)

**Comment by tbrisker** [comment]
Thanks @apuntamb and @ShimShtein !

**Pull request merged**